modules/gitops-configuring-respect-rbac-using-gitops.adoc

```diff
@@ -0,0 +1,100 @@
+
+To enable this feature, set the `.spec.controller.respectRBAC` key in the Argo CD resource. After you set this key, the controller automatically stops watching resources it cannot list or access. For example, this prevents a situation where the Argo CD cluster role restricts Argo CD from watching OpenShift Routes, which would otherwise result in an error during synchronization, stating that it cannot watch the resource.
+
+You can enable the `respectRBAC` feature by creating an Argo CD instance in the CLI or the Web UI. 
+
+.Pre-requisites
+
+Ensure that you created a namespace and updated it in the *Subscription* resource, so it can host a cluster-scoped Argo CD instance. For more information, see Using an Argo CD instance to manage cluster-scoped resources.
+
+[id="configuring-respectRBAC-using-the-cli_{context}"]
+== Configuring respectRBAC using the CLI
+
+You can configure `respectRBAC` feature in the CLI.
+
+.Procedure
+
+. Create a YAML object file, for example, `argo-cd-resource.yaml`, to configure the `respectRBAC` feature using the CLI. 
++
+.Example create an Argo CD instance using the respectRBAC feature
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd #<1>
+spec:
+  controller:
+    respectRBAC: strict #<2>
+----
+<1> Specify the name of the Argo CD instance.
+<2> Specify the value of the `.spec.controller.respectRBAC` key in the Argo CD resource. You can set it to `normal` or `strict`. If the value is set to `strict`, Argo CD performs additional validations of RBAC resources to determine permissions. This setting increases the number of API calls to the server as it is more accurate compared to `normal`. Switch to `strict` if Argo CD reports errors indicating it cannot access resources when the value is set to `normal`. If the value is set to `normal`, Argo CD determines its RBAC permissions by attempting to list each resource. By default it is recommended to start with `normal`, as it balances accuracy and speed because resource listing is a lightweight operation.
+
+. Apply the changes to the YAML object file by running the following command.
++
+[source,terminal]
+----
+$ oc apply -f argocd-resource.yaml -n argo-cd-instance #<1>
+----
+<1> Specify the name of the YAML file that includes the Argo CD resource and the namespace that hosts the Argo CD instance.
++
+. Verify the successful installation of Argo CD by checking the status of the ArgoCD `.status.phase` field. The {gitops-shortname} Operator sets this field to `Available` when the installation is complete. Run the following command to check the status:
++
+[source,terminal]
+----
+oc get argocd <argocd-instance-name> -n <argocd namespace> -o jsonpath='{.status.phase}' #<1>
```

Comment:
```suggestion
$ oc get argocd <argocd_instance_name> -n <argocd_namespace> -o jsonpath='{.status.phase}' #<1>
```

- Per this guideline, we use underscores for variables -- https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values
- Add missing `$`

Reply:
Fixed.